doc/user/sql-grammar/sql-grammar.bnf

```diff
@@ -232,6 +232,7 @@ create_source_mysql ::=
   'FROM' 'MYSQL' 'CONNECTION' connection_name
   ( ( '(' 'TEXT COLUMNS' ('(' (column_name) ( ( ',' column_name ) )* ')')? )? )
       ( ( ',' 'IGNORE COLUMNS' ('(' (column_name) ( ( ',' column_name ) )* ')')? ')' )? )
+      ( ( ',' 'EXCLUDE COLUMNS' ('(' (column_name) ( ( ',' column_name ) )* ')')? ')' )? )
```

Comment:
```suggestion
      ( ( ',' ('IGNORE COLUMNS' | 'EXCLUDE COLUMNS' ) ('(' (column_name) ( ( ',' column_name ) )* ')')? ')' )? )
```

Reply:
Ah sorry, I missed this suggestion. Going to merge this while CI is green, but I'll circle back on this next Thursday when I remove the `IGNORE COLUMNS` from the syntax diagram entirely.